src/record/import/index.ts

```diff
@@ -42,24 +42,33 @@ export const run: (
         ? userSelected(fields, fieldsJson, updateKey)
         : defaultTransformer()
     );
-    const { content, format } = await readFile(filePath, encoding);
-    const records = await parseRecords({
-      source: content,
+    const { format } = openFsStreamWithEncode(filePath, encoding);
```

Comment:
@mshrtsr 

I have one question related to `format`.

Now, we called double `openFsStreamWithEncode`
The first one is only for getting the format
The second one is for generating localRecord
==> Is there any reason that we must validate and extract the format inside the `openFsStreamWithEncode` function?
Can we separate them? If so, we don't need to call it twice.